### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by wkal-pubnub**

### CODE REVIEW

File: src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java [hunk 1]
```diff
@@ -7,8 +7,6 @@
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static com.pubnub.kafka.connect.PubNubKafkaConnectorConfig.*;
```
Comment (marcin-cebo) [inline src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java:11]: Is method `checkingNonRequiredDefaults` used?

File: src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java [hunk 2]
```diff
@@ -7,8 +7,6 @@
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertEquals;
```
Comment (marcin-cebo) [inline src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java:10]: Test that checks that all required params are set might be beneficial so it depict which params are mandatory.
What do you think?

File: src/main/java/com/pubnub/kafka/connect/PubNubKafkaSinkConnectorTask.java [hunk 3]
```diff
@@ -18,6 +20,16 @@
 public class PubNubKafkaSinkConnectorTask extends SinkTask {
     private final Logger log = LoggerFactory.getLogger(this.toString());
 
+    @VisibleForTesting
```
Comment (marcin-cebo) [inline src/main/java/com/pubnub/kafka/connect/PubNubKafkaSinkConnectorTask.java:24]: I am assuming that it is difficult to add PubNub instance to constructor of the class.

**Review by marcin-cebo — APPROVED** [review]

File: src/main/java/com/pubnub/kafka/connect/PubNubKafkaSinkConnectorTask.java (hunk 3 above)
Comment (wkal-pubnub) [reply src/main/java/com/pubnub/kafka/connect/PubNubKafkaSinkConnectorTask.java:24]: changed to constructor

File: src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java (hunk 2 above)
Comment (wkal-pubnub) [reply src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java:10]: added

File: src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java (hunk 1 above)
Comment (wkal-pubnub) [reply src/test/java/com/pubnub/kafka/connect/PubNubConnectorConfigTest.java:11]: no, removed

**Pull request merged**